airbyte-connector-builder/connector_builder/generated/apis/default_api_interface.py

```diff
@@ -0,0 +1,125 @@
+# This file was auto-generated from Airbyte's custom OpenAPI templates
+# coding: utf-8
+
+import inspect
+from abc import ABC, abstractmethod
+from typing import Callable, Dict, List  # noqa: F401
+
+from fastapi import (  # noqa: F401
+    APIRouter,
+    Body,
+    Cookie,
+    Depends,
+    Form,
+    Header,
+    Path,
+    Query,
+    Response,
+    Security,
+    status,
+)
+
+from connector_builder.generated.models.extra_models import TokenModel  # noqa: F401
+
+
+from connector_builder.generated.models.invalid_input_exception_info import InvalidInputExceptionInfo
+from connector_builder.generated.models.known_exception_info import KnownExceptionInfo
+from connector_builder.generated.models.stream_read import StreamRead
+from connector_builder.generated.models.stream_read_request_body import StreamReadRequestBody
+from connector_builder.generated.models.streams_list_read import StreamsListRead
+from connector_builder.generated.models.streams_list_request_body import StreamsListRequestBody
+
+
+class DefaultApi(ABC):
```

Comment:
should we add another warning in the class to be sure we're aware that this is autogenerated? I'm worried the one-liner above the imports is too easy to miss.

Maybe something like [this](https://github.com/airbytehq/airbyte/pull/18886/files#diff-2d9027a45428d469f15ff115a17ff869543905abd0af7d277ec6373c2167eeccR13-R15)

Reply:
fair point, done